### ISSUE TITLE
Xkbcommon failed to build 

### ISSUE THREAD

**Issue by fish4terrisa-MSDSM** [termux/termux-x11#195]
When i build termux-x11,i got this error:
```
C/C++: /data/data/com.termux/files/home/termux-x11/app/src/main/jni/libxkbcommon/xkbcommon/src/atom.c:191:1: error: non-void function does not return a value in all control paths [-Werror,-Wreturn-type]
C/C++: }
C/C++: ^
C/C++: 1 error generated.
C/C++: make: *** [/data/data/com.termux/files/usr/lib/android-ndk-r23b/build/core/build-binary.mk:476: /data/data/com.termux/files/home/termux-x11/app/build/intermediates/cxx/Debug/4m1i2l36/obj/local/arm64-v8a/objs-debug/xkbcommon/xkbcommon/src/atom.o] Error 1

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:buildNdkBuildDebug[arm64-v8a]'.
> com.android.ide.common.process.ProcessException: Android NDK: WARNING:/data/data/com.termux/files/home/termux-x11/app/src/main/jni/lorie/Android.mk:lorie: non-system libraries in linker flags: -lwayland-server
  Android NDK:     This is likely to result in incorrect builds. Try using LOCAL_STATIC_LIBRARIES 
  Android NDK:     or LOCAL_SHARED_LIBRARIES instead to list the library dependencies of the
  Android NDK:     current module
  [arm64-v8a] Compile        : xkbcommon <= atom.c

  C++ build system [build] failed while executing:
      /data/data/com.termux/files/usr/lib/android-ndk-r23b/ndk-build \
        NDK_PROJECT_PATH=null \
        APP_BUILD_SCRIPT=/data/data/com.termux/files/home/termux-x11/app/src/main/jni/Android.mk \
        NDK_APPLICATION_MK=/data/data/com.termux/files/home/termux-x11/app/src/main/jni/Application.mk \
        APP_ABI=arm64-v8a \
        NDK_ALL_ABIS=arm64-v8a \
        NDK_DEBUG=1 \
        APP_PLATFORM=android-24 \
        NDK_OUT=/data/data/com.termux/files/home/termux-x11/app/build/intermediates/cxx/Debug/4m1i2l36/obj \
        NDK_LIBS_OUT=/data/data/com.termux/files/home/termux-x11/app/build/intermediates/cxx/Debug/4m1i2l36/lib \
        lorie \
        xkbcommon
    from /data/data/com.termux/files/home/termux-x11/app
  /data/data/com.termux/files/home/termux-x11/app/src/main/jni/libxkbcommon/xkbcommon/src/atom.c:191:1: error: non-void function does not return a value in all control paths [-Werror,-Wreturn-type]
  }
  ^
  1 error generated.
  make: *** [/data/data/com.termux/files/usr/lib/android-ndk-r23b/build/core/build-binary.mk:476: /data/data/com.termux/files/home/termux-x11/app/build/intermediates/cxx/Debug/4m1i2l36/obj/local/arm64-v8a/objs-debug/xkbcommon/xkbcommon/src/atom.o] Error 1
```
Does anyone know how to fix it?

**Comment by twaik** [comment]
I'll fix it soon...

**Comment by twaik** [comment]
Fixed with [#5b73f8b73f8ff](https://github.com/termux/termux-x11/commit/5b73f8f49713186759c5d66747c207eb3d0b0f32)
